### PR TITLE
[WIP] Support process

### PR DESCRIPTION
**Pull request by chrisblackburn**

### CODE REVIEW

File: guides/process/support/faqs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## When should I mark as pending versus closed?
+
+Pending is for when waiting for a reply which is needed to progress. Closed is for when an issue is resolved/backlogged. Often a user will reply to the final message - this can just be closed unless it requires further action to be taken.
+
+## Who replies to customers, should it be same person or just whoever is on triage that day?
+
+In most cases whoever is on triage that day: outstanding triage work should be handed over from day to day. Occasionally specific domain knowledge may be required in which case it may be easier/more appropriate for that person to respond instead.
+
+## Who does larger quotations/estimation of work that is requested via groove?
```
Comment (davidwinter) [inline guides/process/support/faqs.md]: ~groove~ zendesk

File: guides/process/support/triage.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+On ocassion customers will ask for things via Slack or phone, in which case we can create the ticket on their behalf in Zendesk.
+
+## Process Overview
+
+It is the responsibility of the Triage Pair to:
+
+- [ ] 1. Triage the oldest item from the queue
+- [ ] 2. Record time in Harvest
+- [ ] 3. Ensure we're meeting our Service Level Agreements
```
Comment (davidwinter) [inline guides/process/support/triage.md]: Link to SLA?

File: guides/process/support/triage.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Moving the request to the Customer Roadmap should be considered a successful triage resolution, and so the ticket should be closed.
+
+Customers may ask for time estimates for larger pieces of work. This [FAQ answer](process/support/faqs.md) should help.
+
+- [ ] You've [Popped the Why Stack](http://www.mattblodgett.com/2009/01/pop-stack.html) to ensure you understand what commercial goal the customer is trying to solve
+- [ ] You've successfully resolved the request or added to the Customer Roadmap
+- [ ] You can assert that any code change you made has improved the quality of the application
+
+## 2. Record time in Harvest
```
Comment (davidwinter) [inline guides/process/support/triage.md]: And to potentially start at the 0:15 mark, even if the email takes 5 mins to write?

File: guides/process/support/triage.md (hunk 2 above)
Comment (chrisblackburn) [inline guides/process/support/triage.md]: No formal SLA. I was thinking the acknowledgement in 1 day and resolution in 2 as listed below was enough to start?

Comment (davidwinter) [inline guides/process/support/triage.md]: 👍

**Comment by davidwinter** [comment]
LGTM! 💥

**Pull request merged**